compiler/luci/pass/src/QuantizationUtils.h

```diff
@@ -23,6 +23,9 @@
 namespace luci
 {
 
+// Compute max scale using given data type for symmetric quantization (int4/int8/int16)
+int32_t compute_sym_max_scale(const loco::DataType &type);
```

Comment:
Could you rename the function?
```suggestion
// Return the max value of dtype for symmetric quantization (int4/int8/int16)
int32_t max_for_symm_quant(const loco::DataType type);
```